guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -64,6 +64,18 @@ If an attacker, located in the network between {Project} and a host, fetches the
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 * On the *Advanced* tab, in the *Repositories* field, you can list repositories to be added before the registration is performed.
 You do not have to specify repositories if you provide them in an activation key.
+ifdef::orcharhino[]
+ifdef::debian,ubuntu[]
++
+To verify synchronized {client-content-type} content, you can use the `pulp_deb_signing.key` file on your {SmartProxy} as GPG public key.
+For example, `\https://{foreman-example-com}/pub/pulp_deb_signing.key`.
+endif::[]
+ifndef::debian,ubuntu[]
++
+To verify synchronized {client-content-type} content, you can use {Project} API to get associated GPG public keys of repositories.
```

Comment:
Does Hammer also expose this? If so, that's even nicer.

Reply:
You can find out which GPG key is associated to a repository via Hammer CLI, yes. However, you require authentication for that. For the Katello API, you do not need to provide any credentials. This makes it very useful for host registration. Examples:

* `hammer content-credentials info --id 1 --organization-id 1`
* `hammer repository info --id 24 --organization-id 1`
* https://foreman-katello.example.com/katello/api/v2/repositories/24/gpg_key_content